tools/Dashboard/DevHome.Dashboard/Helpers/WidgetServiceHelper.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using Windows.Management.Deployment;
+
+namespace DevHome.Dashboard.Helpers;
+internal class WidgetServiceHelper
+{
+    private readonly Version minSupportedVersion400 = new (423, 3800);

```

Comment:
What are these numbers?  OS versions?

Reply:
Minimum versions of the experience pack that have what we need.